BasicSample/ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2020.3.28f1
-m_EditorVersionWithRevision: 2020.3.28f1 (f5400f52e03f)
+m_EditorVersion: 2020.3.31f1
+m_EditorVersionWithRevision: 2020.3.31f1 (6b54b7616050)
```

Comment:
is this the recommended version of unity for this project?

Reply:
it's the latest that works for HL2.